modules/@apostrophecms/login/index.js

```diff
@@ -889,8 +889,8 @@ module.exports = {
         };
       },
 
-      async clearLoginAttempts (username, namespace = loginAttemptsNamespace) {
-        await self.apos.cache.delete(username, namespace);
+      async clearLoginAttempts (namespace = loginAttemptsNamespace, username) {
```

Comment:
Defaults for the first argument don't work if there is a second argument, which is required here

Also, tests are failing

Reply:
I'm wrong about defaults not being supported for first arguments of several, if you pass undefined they are used